spec/models/friendship_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Friendship, type: :model do
+  let(:user) { create(:user) }
+  let(:user2) { create(:user) }
+  let(:friendship) { create(:friendship) }
+
+  describe 'validations' do
```

Comment:
When we are validating a model, we shouldn't test an already `created` test data. For instance here, instead of `create(:friendship)` we should have run `build(:friendship)`